package.json

```diff
@@ -65,7 +65,7 @@
     "react-native-safe-area": "^0.4.1",
     "react-native-screens": "^1.0.0-alpha.23",
     "react-native-simple-toast": "^1.0.0",
-    "react-native-sound": "^0.10.9",
+    "react-native-sound": "~0.10.9",
```

Comment:
I think the existing `^0.10.9` already has the same meaning.

You can try it empirically here:
https://semver.npmjs.com/

Reply:
Ah OK! I'd mistaken the caret's behavior here as the caret's behavior if the leftmost spot wasn't a zero.

From a relevant-looking [doc](https://docs.npmjs.com/misc/semver):
> Caret Ranges ^1.2.3 ^0.2.5 ^0.0.4
>
> Allows changes that do not modify the left-most non-zero digit in the `[major, minor, patch]` tuple.